scripts/superdesk/keyboard/keyboard.js

```diff
@@ -332,6 +332,35 @@
         };
     }])
 
+    .directive('sdHotkey', ['keyboardManager', '$timeout', function (keyboardManager, $timeout) {
+            return {
+                link: function (scope, elem, attrs, ctrl) {
+                    var hotkey = attrs.sdHotkey,
+                        callback = scope.$eval(attrs.sdHotkeyCallback),
+                        options = scope.$eval(attrs.sdHotkeyOptions);
+
+                    keyboardManager.bind(hotkey, function (e) {
+                        e.preventDefault();
+                        if (callback) {
+                            callback();
+                        } else {
+                            elem.click();
+                        }
+                    }, options);
```

Comment:
when it's global (`options.global == true`), there should be `unbind` on `scope.$destroy`, otherwise there is memory leak when navigating away from workspace and back